src/PhpSpreadsheet/Writer/Xls/Worksheet.php

```diff
@@ -772,6 +788,7 @@ private function writeBoolErr($row, $col, $value, $isError, $xfIndex)
      * Returns  0 : normal termination
      *         -1 : formula errors (bad formula)
      *         -2 : row or column out of range
+     *         -3 : parse raised exception, probably due to definedname
      *
```

Comment:
Looking to improving the readability of code going forward, these numeric values would be better as constants with meaningful names

Reply:
Newly added lines 789-793 define constants, lines 799-802 describe them, and lines 444, 860, 883, and 885 use them.